immutable-collection-encoding-test/pom.xml

```
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hubspot.immutables</groupId>
+    <artifactId>hubspot-immutables</artifactId>
+    <version>1.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>immutable-collection-encoding-test</artifactId>
+
+  <properties>
+    <basepom.check.skip-findbugs>true</basepom.check.skip-findbugs>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.errorprone</groupId>
+        <artifactId>error_prone_annotations</artifactId>
+        <version>2.1.3</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.hubspot</groupId>
+      <artifactId>immutable-collection-encoding</artifactId>
```

I think this can be `provided` too. Same for the root pom and `hubspot-style` too.
```suggestion
      <artifactId>immutable-collection-encoding</artifactId>
      <scope>provided</scope>
```

---

so I don't this actually works b.c we need a runtime dep on the encoding module for the `BuilderState` enum (ideally I could get the encoding to copy this over into the builder itself, but ATM that does not work). We don't ever have people put `hubspot-style` at scope provided either (not totally sure for the reason on that one) so I am not worried about being unable to use `provided` for the encoding.